### DISCUSSION TITLE
Death Charge Cool down Notification

### DISCUSSION THREAD

**Discussion by GaryGooch** [runelite/runelite#14976] [Q&A] [answered]
Currently there is a 60 second timer that is displayed as an info box for Death charge (Arceeus spell that gives special attack energy back if you kill and npc whilst it's active).

But once it had procced after activation, there is no indicator that the spell can be recast once it had come off cool down without physically checking the spellbook and seeing if the icon is no longer slightly faint in colour.

Ideally a info box timer to show when it can be recast or a notification in the chat box would be ideal.

**Comment by Enriath** [comment, marked as answer]
There is already an option for `Arceuus spells cooldown` in the `Timers` plugin
